deployments/kovan/migrations/1644388553_deploy_kovan.ts

```diff
@@ -22,135 +24,227 @@ let cloneAddr = {
+  const { ethers } = deploymentManager.hre;
+  let signer = await deploymentManager.getSigner();
+  let signerAddress = signer.address;
+
+  let usdcProxyAdminArgs: [] = [];
+  let usdcProxyAdmin = await deploymentManager.deploy<ProxyAdmin, ProxyAdmin__factory, []>(
+    'vendor/proxy/transparent/ProxyAdmin.sol',
+    usdcProxyAdminArgs
+  );
+
+  let fauceteer = await deploymentManager.deploy<Fauceteer, Fauceteer__factory, []>(
+    'test/Fauceteer.sol',
+    []
+  );
+
+  let usdcImplementation = await deploymentManager.clone(
+    cloneAddr.usdcImplementation,
+    [],
+    cloneNetwork
+  );
+
+  let usdc;
+  let usdcProxy = await deploymentManager.clone(
+    cloneAddr.usdcProxy,
+    [usdcImplementation.address],
+    cloneNetwork
+  );
+
+  debug(`Changing admin of USDC proxy to ${usdcProxyAdmin.address}`);
+  await deploymentManager.asyncCallWithRetry(
+    (signer_) => wait(usdcProxy.connect(signer_).changeAdmin(usdcProxyAdmin.address))
+  )
+  usdc = usdcImplementation.attach(usdcProxy.address);
+  // Give signer 10,000 USDC
+  debug(`Initializing USDC`);
+  await deploymentManager.asyncCallWithRetry(
+    (signer_) => wait(
+      usdc.connect(signer_).initialize(
+        'USD Coin',
+        'USDC',
+        'USD',
+        6,
+        signerAddress,
+        signerAddress,
+        signerAddress,
+        signerAddress
+      )
+    )
+  );
+
+  let wbtc = await deploymentManager.clone(
+    cloneAddr.wbtc,
+    [],
+    cloneNetwork
+  );
+
+  let weth = await deploymentManager.clone(
+    cloneAddr.weth,
+    [],
+    cloneNetwork
+  );
+  // Give admin 0.01 WETH tokens [this is a precious resource here!]
+  debug(`Minting some WETH`);
+  await deploymentManager.asyncCallWithRetry(
+    (signer_) => wait(weth.connect(signer_).deposit({ value: exp(0.01, 18) }))
+  );
+
+  let comp = await deploymentManager.clone(
+    cloneAddr.comp,
+    [signerAddress],
+    cloneNetwork
+  );
+
+  const blockNumber = await ethers.provider.getBlockNumber();
+  const blockTimestamp = (await ethers.provider.getBlock(blockNumber)).timestamp;
+
+  let uni = await deploymentManager.clone(
+    cloneAddr.uni,
+    [signerAddress, signerAddress, blockTimestamp + 100],
+    cloneNetwork
+  );
+
+  let link = await deploymentManager.clone(
+    cloneAddr.link,
+    [],
+    cloneNetwork
+  );
+
+  // Contracts referenced in `configuration.json`.
+  let contracts = new Map<string, Contract>([
+    ['USDC', usdc],
+    ['WBTC', wbtc],
+    ['WETH', weth],
+    ['COMP', comp],
+    ['UNI', uni],
+    ['LINK', link],
+  ]);
+
+  // Deploy all Comet-related contracts
+  let { cometProxy, configuratorProxy, timelock, rewards } = await deployNetworkComet(
+    deploymentManager,
+    { all: true },
+    {},
+    contracts
+  );
+
+  // Deploy Bulker
+  const bulker = await deploymentManager.deploy<Bulker, Bulker__factory, [string, string, string]>(
+    'Bulker.sol',
+    [timelock.address, cometProxy.address, weth.address]
+  );
```

Comment:
I guess bulker is the main thing stopping the interface being 'return the args to `deployComet`, so I guess I'll start with interface being return `Roots` (for separate `deploy` that is)